Podfile

```diff
@@ -154,7 +154,8 @@ target 'WordPress' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.46.0'
+    #gutenberg  :tag => 'v1.46.0'
+    gutenberg  :commit => 'dcacde'
```

Comment:
I undid this change temporarily on my machine, ran `rake xcode` and ran the project – without the packager running – expecting to see the red error screen, and I didn't get an error when I opened the story post.

Reply:
I didn't get an error either when running the packager on the Gutenberg Mobile branch, which is a good thing, but I expected to be able to reproduce the error by undoing the above change but wasn't successful.